IANotificationsTestProject/In App Notifications/NoticeView.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+
+class NoticeView: UIView {
+    
+    // MARK: Properties
+
+    @IBOutlet weak var stackView: UIStackView!
+    @IBOutlet weak var noticeLabel: UILabel!
+    @IBOutlet weak var noticeButton: UIButton!
+
+    var delegate: NoticePresentingDelegate?
+    var action: (() -> Void)? {
+        didSet {
+            noticeButton.isHidden = action == nil ? true : false
```

Comment:
```suggestion
            noticeButton.isHidden = action == nil
```

Reply:
The `action == nil` evaluation yields a boolean (no need for the ternary operator)